core/src/subgraph/instance_manager.rs

```diff
@@ -406,18 +406,19 @@ where
         let instance =
             SubgraphInstance::from_manifest(&logger, manifest, host_builder, host_metrics.clone())?;
 
+        let inputs = IndexingInputs {
+            deployment: deployment.clone(),
+            features,
+            start_blocks,
+            store,
+            triggers_adapter,
+            chain,
+            templates,
+            unified_api_version,
+        };
```

Comment:
the ctx needs to be mut borrowed at some point and this doesn't allow to re-use it unless it implements Clone or something like that. I thought splitting it was the best option because it is read-only and this way we don't need to clone it everywhere.